### ISSUE TITLE
test_Combine_run failed on Fedora

### ISSUE THREAD

**Issue by sagitter** [sbmlteam/libCombine#24]
Hello.

`zipper` and `libCombine` are needed for compiling `COPASI-4.23` on Fedora, so these libraries are tested on Fedora systems before including them as new packages.

`libCombine` test is failing with following error:

```
+ ctest --force-new-ctest-process -VV
UpdateCTestConfiguration  from :/builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/build/DartConfiguration.tcl
Parse Config file:/builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/build/DartConfiguration.tcl
Parse Config file:/builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/build/DartConfiguration.tcl
Parse Config file:/builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/build/DartConfiguration.tcl
Parse Config file:/builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/build/DartConfiguration.tcl
Test project /builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/build
Test project /builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/build
Constructing a list of tests
Constructing a list of tests
Done constructing a list of tests
Done constructing a list of tests
Updating test list for fixtures
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph...
Checking test dependency graph end
Checking test dependency graph end
test 1
test 1
    Start 1: test_Combine_run
    Start 1: test_Combine_run
1: Test command: /builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/build/Combine-test
1: Test command: /builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/build/Combine-test
1: Environment variables: 
1: Environment variables: 
1:  srcdir=/builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/src/test
1:  srcdir=/builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/src/test
1: Test timeout computed to be: 1500
1: Test timeout computed to be: 1500
1: 
1: 
1: ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1: ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1: Combine-test is a Catch v1.5.6 host application.
1: Combine-test is a Catch v1.5.6 host application.
1: Run with -? for options
1: Run with -? for options
1: 
1: 
1: -------------------------------------------------------------------------------
1: -------------------------------------------------------------------------------
1: Scenario: creating a new combine archive
1: Scenario: creating a new combine archive
1:      Given: a user creates a new archive
1:      Given: a user creates a new archive
1:       When: a document is added
1:       When: a document is added
1:       Then: the file can be found in the archive
1:       Then: the file can be found in the archive
1:   And when: metadata is added
1:   And when: metadata is added
1:       Then: the metadata is accessible
1:       Then: the metadata is accessible
1:   And when: the archive is saved
1:   And when: the archive is saved
1:       Then: it can be loaded and the files are present.
1:       Then: it can be loaded and the files are present.
1: -------------------------------------------------------------------------------
1: -------------------------------------------------------------------------------
1: /builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/src/test/combine_test_create.cpp:54
1: /builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/src/test/combine_test_create.cpp:54
1: ...............................................................................
1: ...............................................................................
1: 
1: 
1: /builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/src/test/combine_test_create.cpp:126: FAILED:
1: /builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/src/test/combine_test_create.cpp:126: FAILED:
1:   REQUIRE( second.getManifest()->getNumContents() == 1 )
1:   REQUIRE( second.getManifest()->getNumContents() == 1 )
1: with expansion:
1: with expansion:
1:   2 == 1
1:   2 == 1
1: 
1: 
1: -------------------------------------------------------------------------------
1: -------------------------------------------------------------------------------
1: Scenario: getting / setting metadata elements
1: Scenario: getting / setting metadata elements
1:      Given: A new vcard and description is is created.
1:      Given: A new vcard and description is is created.
1:       When: the description is changed
1:       When: the description is changed
1:       Then: the description values are updated
1:       Then: the description values are updated
1:   And when: the vcard is changed
1:   And when: the vcard is changed
1:        And: the vcard is updated
1:        And: the vcard is updated
1:   And when: the vcard is added to the description
1:   And when: the vcard is added to the description
1:       Then: the descriptio contains the vcard
1:       Then: the descriptio contains the vcard
1:   And when: the description is written to file and read back in
1:   And when: the description is written to file and read back in
1:       Then: the values remain the same
1:       Then: the values remain the same
1: -------------------------------------------------------------------------------
1: -------------------------------------------------------------------------------
1: /builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/src/test/metadata_test_create.cpp:17
1: /builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/src/test/metadata_test_create.cpp:17
1: ...............................................................................
1: ...............................................................................
1: 
1: 
1: /builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/src/test/metadata_test_create.cpp:75: FAILED:
1: /builddir/build/BUILD/libCombine-f904decd5d863b02b187f13e53457785f562ce21/src/test/metadata_test_create.cpp:75: FAILED:
1:   REQUIRE( descs.size() == 1 )
1:   REQUIRE( descs.size() == 1 )
1: with expansion:
1: with expansion:
1:   0 == 1
1:   0 == 1
1: 
1: 
1: ===============================================================================
1: ===============================================================================
1: test cases:   5 |   3 passed | 2 failed
1: test cases:   5 |   3 passed | 2 failed
1: assertions: 103 | 101 passed | 2 failed
1: assertions: 103 | 101 passed | 2 failed
1: 
1: 
1/1 Test #1: test_Combine_run .................***Failed    0.03 sec
1/1 Test #1: test_Combine_run .................***Failed    0.03 sec
0% tests passed, 1 tests failed out of 1
0% tests passed, 1 tests failed out of 1
Total Test time (real) =   0.03 sec
Total Test time (real) =   0.03 sec
The following tests FAILED:
The following tests FAILED:
	  1 - test_Combine_run (Failed)
	  1 - test_Combine_run (Failed)
BUILDSTDERR: Errors while running CTest
BUILDSTDERR: Errors while running CTest
BUILDSTDERR: error: Bad exit status from /var/tmp/rpm-tmp.oG9UVR (%check)
BUILDSTDERR: error: Bad exit status from /var/tmp/rpm-tmp.oG9UVR (%check)
RPM build errors:
RPM build errors:
BUILDSTDERR:     Bad exit status from /var/tmp/rpm-tmp.oG9UVR (%check)
BUILDSTDERR:     Bad exit status from /var/tmp/rpm-tmp.oG9UVR (%check)
Child return code was: 1
Child return code was: 1
EXCEPTION: [Error()]
```

Used CMake-3.11.0.
Full build log: https://copr-be.cloud.fedoraproject.org/results/sagitter/ForTesting/fedora-rawhide-x86_64/00743739-libCombine/build.log.gz

**Comment by fbergmann** [comment]
thanks for letting me know ... i will get that sorted as soon as i can ... is it rawhide ?

**Comment by sagitter** [comment]
>  is it rawhide ?

Yes, but it happens on other branches too.

**Comment by fbergmann** [comment]
thanks for letting me know i've now resolved that issue.

**Comment by sagitter** [comment]
Fixed in 8902b68d00ebf1f915b7694a64d65d7e14cf2117 commit.
Thanks.
